src/plugins/visualizations/public/wizard/new_vis_modal.tsx

```diff
@@ -84,6 +85,19 @@ class NewVisModal extends React.Component<TypeSelectionProps, TypeSelectionState
       return null;
     }
 
+    if (this.props.newVisType) {
```

Comment:
I am so sorry but I dont get what we want to do. What kind of information has the newVisType prop. And why do we return null if it has a value?

Reply:
So it turns out I didn't actually need to go through the new vis modal to add these visualizations, and instead use the embeddable state transfer service. I removed all the changes to the new vis modal.